pages/index.vue

```diff
@@ -85,9 +85,14 @@ const totalAPICount = computed(() => webApiList.value.length)
 function updateMode(newValue: DisplayMode) {
   displayMode.value = newValue
 }
-useTestWebApis()
 
-watch(() => displayMode.value, (newVal, oldVal) => {
-  console.log(`DisplayMode changed from '${oldVal}' to '${newVal}'`)
+// watch(() => displayMode.value, (newVal, oldVal) => {
+//   console.log(`DisplayMode changed from '${oldVal}' to '${newVal}'`)
+// })
+
+onMounted(() => {
```

Comment:
Running this only onMounted will leave us without SSR tho

Reply:
With this https://github.com/toniengelhardt/webapicheck/pull/28#issuecomment-1438582395 we can fix the prerendering issue and get SSR when there's a query param and populating the fields on the client when there's not.